test/IIS.FunctionalTests/Inprocess/StdOutRedirectionTests.cs

```diff
@@ -0,0 +1,218 @@
+// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using System;
+using System.IO;
+using System.Linq;
+using System.Threading.Tasks;
+using Microsoft.AspNetCore.Server.IIS.FunctionalTests.Utilities;
+using Microsoft.AspNetCore.Server.IISIntegration.FunctionalTests;
+using Microsoft.AspNetCore.Server.IntegrationTesting;
+using Microsoft.AspNetCore.Server.IntegrationTesting.IIS;
+using Microsoft.AspNetCore.Testing.xunit;
+using Xunit;
+
+namespace IIS.FunctionalTests.Inprocess
+{
+    [Collection(PublishedSitesCollection.Name)]
+    public class StdOutRedirectionTests : IISFunctionalTestBase
+    {
+        private readonly PublishedSitesFixture _fixture;
+        public StdOutRedirectionTests(PublishedSitesFixture fixture)
+        {
+            _fixture = fixture;
+        }
+
+        [ConditionalFact]
+        [SkipIfDebug]
+        public async Task FrameworkNotFoundExceptionLogged_Pipe()
+        {
+            var deploymentParameters = GetStartupExceptionParameters();
+
+            var deploymentResult = await DeployAsync(deploymentParameters);
+
+            InvalidateRuntimeConfig(deploymentResult);
+
+            var response = await deploymentResult.HttpClient.GetAsync("/");
+            Assert.False(response.IsSuccessStatusCode);
+
+            StopServer();
+
+            EventLogHelpers.VerifyEventLogEvent(deploymentResult, TestSink,
```

Comment:
We want to verify that `The specified framework 'Microsoft.NETCore.App', version '2.9.9' was not found` was written to event log.

Reply:
We may need to change debug log to replace newlines in event log messaged with some other character to keep them on one line.

Reply:
> We want to verify that The specified framework 'Microsoft.NETCore.App', version '2.9.9' was not found was written to event log.
Like the actual event log?

Reply:
No, using whatever infrastructure you have. You look for `The specified framework 'Microsoft.NETCore.App', version '2.9.9' was not found ` directly in the logsink and it might get there even if it wasn't event logged.

Reply:
Right, but you don't check `The specified framework 'Microsoft.NETCore.App', version '2.9.9' was not found` with event log helper.